### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by hluk**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 MAINTAINER Pavlina Bortlova <pbortlov@redhat.com>
 
-LABEL description="Review-rot - gather information about opened merge or pull requests"
-LABEL summary="review-rot git gitlab github pagure gerrit"
-LABEL vendor="PnT DevOps Automation - Red Hat, Inc."
+LABEL \
+    description="Review-rot - gather information about opened merge or pull requests" \
+    summary="review-rot git gitlab github pagure gerrit" \
+    vendor="PnT DevOps Automation - Red Hat, Inc."
```
Comment (pbortlov) [inline Dockerfile:7]: sorry for bothering: could you plase change the vendor? to Process-tools guild? https://docs.engineering.redhat.com/display/EXD/reviewrot

**Comment by hluk** [comment]
I've set up automated builds in quay from my repo clone -- I do not have a permission to do it for repos in redhat-aqe org.

I've also tested the new container and review-rot works properly.

BTW, the executable name in container is now `review-rot` instead of `reviewrot` - it might be a good idea to add a symlink to keep it compatible (I previously used https://quay.io/pbortlov/reviewrot/).

File: Dockerfile (hunk 1 above)
Comment (hluk) [reply Dockerfile]: Changed, but not sure which format would you prefer.

**Review by mkosiarc — APPROVED** [review]

**Comment by hluk** [comment]
Can you merge this?

**Comment by mkosiarc** [comment]
@pbortlov could you please check the requested changes and approve this PR?

File: Dockerfile [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
-ADD . /reviewrot
+COPY . /reviewrot
 
 # install review-rot
-RUN pip install --upgrade pip setuptools && python setup.py install
+RUN cd /reviewrot \
+    && pip3 install --upgrade pip setuptools \
+    && python3 setup.py install
+
+# clean up
```
Comment (pbortlov) [inline Dockerfile:29]: Why do we clean up the installed libs? Could you add a comment?

Comment (hluk) [reply Dockerfile]: Comment updated.

**Pull request merged**